<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="Curve_Element" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="nombre_list">
        <property name="styleSheet">
         <string notr="true">color:blue;</string>
        </property>
        <property name="text">
         <string>nombre</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="visibilidad_list">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(88, 224, 173);
border-radius:15px;
border-style: outset;
border-width: 2px;
border-color: black;</string>
        </property>
        <property name="text">
         <string>visibilidad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="color_list">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(88, 224, 173);
border-radius:15px;
border-style: outset;
border-width: 2px;
border-color: black;</string>
        </property>
        <property name="text">
         <string>color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="datos_list">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(88, 224, 173);
border-radius:15px;
border-style: outset;
border-width: 2px;
border-color: black;</string>
        </property>
        <property name="text">
         <string>datos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="borrar_list">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(88, 224, 173);
border-radius:15px;
border-style: outset;
border-width: 2px;
border-color: black;</string>
        </property>
        <property name="text">
         <string>Borrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
